app/ai/gemini.py

```
@@ -0,0 +1,88 @@
+    model_name = model or os.getenv("GEMINI_MODEL") or "gemini-2.5-flash"
+    generation_config = {
+        "temperature": temperature,
+        "max_output_tokens": max_output_tokens,
+    }
+    model_client = genai.GenerativeModel(model_name=model_name, generation_config=generation_config)
+    parts = [prompt]
+    if system:
+        parts = [f"SYSTEM:\n{system}\n\nUSER:\n{prompt}"]
+    try:
+        resp = model_client.generate_content(parts)
+        return resp.text or ""
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)